docs/payment/stripe.md

```diff
@@ -0,0 +1,49 @@
+# Stripe setup guide
+
+In this guide we'll setup [Stripe](https://stripe.com) payment gateway with Saleor Checkout.
+
+## Setup
+
+1. [Sign up for a Stripe account](https://dashboard.stripe.com/register)
+
+2. Make sure you've enabled at least 1 payment provider in Mollie dashboard (Settings > Payment methods):
```

Comment:
```suggestion
2. Make sure you've enabled at least 1 payment method in the Stripe dashboard (Settings > Payment methods):
```

I know this is following the same structure as other guides, so I suggest changing the "payment provider" to "payment method" across all of them. Using "payment provider" is confusing here (for me, Stripe is the payment provider).